src/SqlPersistence.Tests/Outbox/OutboxPersisterTests.cs

```diff
@@ -148,4 +149,63 @@ public void StoreAndGet()
         return await persister.Get(messageId, null);
     }
 
+    [Test]
+    public async Task StoreAndCleanup()
+    {
+        using (var connection = await dbConnection.OpenConnection())
+        {
+            for (var i = 0; i < 13; i++)
```

Comment:
well i wanted more than 1 so multiple would be cleaned up. so any number greater than 1 would do